content/topics/coding-horrors/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Coding Horrors
+ready: true
+---
+
+# Common TDD pitfalls.
```

Comment:
why not just put this on the tdd TOPIC (yes, we have one). So far this content is general enough that it can just be introduced when TDD is introduced.

Always look at the existing content and see if it makes sense to improve something that already exists or make something new. 

Aim for cohesion